samples/WOCCatalog/WOCCatalog/ImagesViewController.m

```diff
@@ -38,6 +40,180 @@ + (UIImage*)scaleImage:(CGImageRef)imageRef scaledRect:(CGRect)rect quality:(CGI
     return scaledImage;
 }
 
++ (void)initCGImageFormat:(CGImageRef)imageRef formatInfo:(vImage_CGImageFormat*)formatInfo {
```

Comment:
If format was an enum or another dword sized value, then I figure a 'get' prefix would be appropriate.  However, in this case all values in a multi-element structure is being set so I feel that an 'init' prefix would be more appropriate here.